modules/react/checkbox/stories/examples/Inverse.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import {Checkbox} from '@workday/canvas-kit-react/checkbox';
 import {Flex} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const styleOverrides = createStyles({
+  gap: system.space.x4,
+  backgroundColor: base.blueberry400,
```

Comment:
for things that use blueberry400, do we want to change this to be `brand.primary`?

Reply:
reference Raisa's comment above

Reply:
yeah, there is more other instances below than I commented

Reply:
I would say default would be using `system.color.primary.default` then when it's branded, we would use `brand`